app/routes/repo/config.js

```diff
@@ -0,0 +1,5 @@
+import TravisRoute from 'travis/routes/basic';
+
+export default TravisRoute.extend({
+  titleToken: 'Trigger Build',
```

Comment:
The page title doesn't match the route name, which could be confusing. I'd suggest thinking about better title once we agree on the route name.

Reply:
i agree, sounds good.

Reply:
As long as the decision is made, would it make sense to make page title some thing like "Repository config"? What do you think @svenfuchs?

Reply:
i've now made it `Repository build config`: https://github.com/travis-ci/travis-web/pull/2399/commits/a1c622350580d3e86212106a5a5f2710cc89b1b0

Reply:
Neat 👍